apps/governance/src/routes/proposals/proposals/Proposals.graphql

```diff
@@ -43,6 +43,31 @@ fragment UpdateMarketStates on Proposal {
   }
 }
 
+fragment UpdateReferralPrograms on Proposal {
+  terms {
+    change {
+      ... on UpdateReferralProgram {
+        changes {
+          id
+          version
```

Comment:
`id` and `version` are internal values, and shouldn't be needed. It's a bug from the gql schema.

Reply:
ok thanks @ValentinTrinque